gpflow/kernels/stationaries.py

```diff
@@ -53,13 +53,19 @@ def __init__(
             slice of indices which controls which columns of X are used (by
             default, all columns are used).
         """
+
+        self._lengthscale_transform = kwargs.pop("lengthscale_transform", None)
```

Comment:
Probably clearer to add this as an explicit, typed optional argument, rather than incorporate it into kwargs and then immediately pop it?

Either way, can you also update the docstring, as well as RELEASE.md with a brief description?